build/charts/nephe/README.md

```diff
@@ -0,0 +1,31 @@
+# nephe
+
+![Version: 0.4.0-dev](https://img.shields.io/badge/Version-0.4.0--dev-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: latest](https://img.shields.io/badge/AppVersion-latest-informational?style=flat-square)
```

Comment:
For the version field, do we need the -dev prefix?

Reply:
its fetch from VERSION file and main has -dev. So its fine